addons/l10n_mt/data/template/account.account-mt.csv

```diff
@@ -0,0 +1,480 @@
+"id","code","name","account_type","reconcile"
```

Comment:
Don't we want the translations in Maltese?

Reply:
Matles use mainly english for legal document from what i have seen online so i think we can let all the localisation in english :smile:

Reply:
If there is no other language supported by us, no need to have the `.pot` then...